docs/benchmarks.md

```diff
@@ -435,9 +435,9 @@ NVBENCH_BENCH(my_cpu_benchmark)
   .set_is_cpu_only(true); // Mark as CPU-only.
 ```
 
-The optional `nvbench::exec_tag::no_gpu` hint may be used to reduce tbe compilation time and
+The optional `nvbench::exec_tag::no_gpu` hint may be used to reduce the compilation time and
 binary size of CPU-only benchmarks. An error is emitted at runtime if this tag is used while
-`is_cpu_only` is false.
+`set_is_cpu_only` is false.
```

Comment:
ah, `is_cpu_only` is the name of the property. feel free to scrap these changes.